scripts/stacklight_infra_install.sh

```diff
@@ -7,5 +7,10 @@ salt -C 'I@influxdb:server' state.sls influxdb -b 1
 salt -C 'I@kibana:server' state.sls kibana.server -b 1
 salt -C 'I@grafana:server' state.sls grafana.server -b 1
 salt -C 'I@nagios:server' state.sls nagios -b 1
+salt -C 'I@elasticsearch:client' state.sls elasticsearch.client.service
+salt -C 'I@elasticsearch:client' service.restart salt-minion --async
```

Comment:
`--async` is an option of the "salt" command. To avoid confusion I would place it before `service.restart` on the command line.